.size-limit.js

```diff
@@ -7,6 +7,25 @@ module.exports = [
     gzip: true,
     limit: '90 KB',
   },
+  {
+    name: '@sentry/browser (incl. Tracing, Replay) - Webpack with treeshaking flags (gzipped)',
+    path: 'packages/browser/build/npm/esm/index.js',
+    import: '{ init, Replay, BrowserTracing }',
+    gzip: true,
+    limit: '90 KB',
+    modifyWebpackConfig: function (config) {
+      const webpack = require('webpack');
+      config.plugins.push(
+        new webpack.DefinePlugin({
+          __SENTRY_DEBUG__: false,
+          __RRWEB_EXCLUDE_CANVAS__: true,
+          __RRWEB_EXCLUDE_SHADOW_DOM__: true,
```

Comment:
Hmm kind of unfortunate that we have inverse semantics here to the `__SENTRY_DEBUG__` flag but i actually prefer the verbose `EXCLUDE` naming pattern.

Reply:
yeah, I was/am also thinking about this - but IMHO for an opt-in behavior it's nicer to set this to `true` if I want to shake this out... 🤔  Maybe we can think about this for `__SENTRY_DEBUG__` in v8 as well, make this `__SENTRY_STRIP_DEBUG__: true` or something like this..